onnxruntime/core/graph/graph.cc

```diff
@@ -842,6 +842,10 @@ Graph::Graph(const Model& owning_model,
     NodeArg* matching_graph_input = GetNodeArg(tensor.name());
     TypeProto t{TypeProtoFromTensorProto(tensor)};
 
+    if (t.tensor_type().elem_type() == ONNX_NAMESPACE::TensorProto_DataType_UNDEFINED) {
```

Comment:
use utils::HasElemType() to maintain compat with protobuf 3

Reply:
Done.